cpp/oneapi/dal/algo/jaccard/backend/cpu/vertex_similarity_default_kernel.cpp

```diff
@@ -224,6 +249,7 @@ size_t intersection(NodeID_t *neigh_u, NodeID_t *neigh_v, NodeID_t n_u, NodeID_t
     }
 
     while (i_u <= (n_u - 4) && i_v <= (n_v - 4)) { // not in last n%8 elements
```

Comment:
comment is similar to one above - what are the naming conventions for variables in the implementation?

Reply:
naming will be fixed by Alexey, here is the draft kernel version